screens/chat/ChatScreen.tsx

```diff
@@ -210,18 +246,30 @@ const ChatScreen = ({ route }: ChatScreenProps) => {
           <WordInfoCard selectedWord={selectedWord} onDismiss={onSelectedWordDismiss} />
         </View>
       </Modal>
-      <View style={styles.header}>
-        <ChatHeader />
-      </View>
+      <CopilotStep
+        name="chat-screen"
+        order={1}
+        text={`This is the chat screen, you are talking with "${
+          currentBot?.name || 'a chatbot'
+        }". You can ask questions or just chat with it.`}
```

Comment:
"or just chat with it" -> "or just chat about anything" OR "or just chat" OR some variation you deem fit